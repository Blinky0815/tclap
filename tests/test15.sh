#!/bin/sh

# failure
../examples/test3  -s bbb homer marge bart -- -hv two > tmp.out 2>&1

if cmp -s tmp.out test15.out; then
	exit 0
else 
	exit 1
fi

