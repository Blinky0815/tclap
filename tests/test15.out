[  ] 0  marge
[  ] 1  bart
[  ] 2  -hv
[  ] 3  two
for float we got : 0
for int we got : 0
for string we got : bbb
for ulabeled we got : homer
for bool A we got : 0
for bool B we got : 0
for bool C we got : 0
